en/components/radial-gauge.md

```diff
@@ -24,89 +24,59 @@ The Radial Gauge provides various overrides that control the shape of the needle
 
```

Comment:
The images above can be replaced with a single one showing just the needles to save some space